config/render/deployment.yaml

```diff
@@ -0,0 +1,835 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
```

Comment:
This namespace is wrong, can you please change it in config/manager/manager.yaml ?

Reply:
Done